src/style.css

```
@@ -1 +1,76 @@
-/* Styles go here */
+html {
+  scroll-behavior: smooth;
+  --header-height: 60px;
+  --text-color: #00acdc;
+  --link-height: 60px;
+}
+
+body {
+  margin: 0;
+  background-color: rgb(106, 134, 192);
+}
+
```

Add this styles in order to remove the scroll on the header and the menu will fit on the screen
```suggestion
}
* {
  box-sizing: border-box;
}
```

---

Извините, так что мне стоит сделать либо исправить?